themes/edx.org/lms/templates/dashboard.html

```diff
@@ -141,29 +142,37 @@
 <section class="dashboard" id="dashboard-main">
 <main class="main-container" id="main" aria-label="Content" tabindex="-1">
   <%
-    from openedx.core.djangoapps.demographics.api.status import show_user_demographics
+    from openedx.core.djangoapps.demographics.api.status import (
+        show_user_demographics, show_call_to_action_for_user
+    )
     from openedx.core.djangoapps.enrollments.api import serialize_enrollments
     serialized_course_enrollments = serialize_enrollments(course_enrollments)
     demographics_enabled = getattr(settings, 'ENABLE_DEMOGRAPHICS_COLLECTION', False)
     display_demographics_banner = demographics_enabled and show_user_demographics(
       user, enrollments=serialized_course_enrollments, entitlements=course_entitlements
-    )
+    ) and show_call_to_action_for_user(user)
```

Comment:
nit: The formatting here looks a little odd. It's easier to read if all the content of the condition is a level indented. Like
```
var = (
    ...
    ...
)
```